tests/test_utils.py

```diff
@@ -298,11 +306,536 @@ def check_multistream_classic(fnames):
+                                science_test_data_dir,
+                                compare_data,
+                                result_dir,
+                                plot_dir,
+                                plots_to_make)
+    else:
+        raise ValueError("this option %s has not been implemented in the \
+                         VIC 5.0 science test suite" % test_type)
+
+
+def plot_snotel_comparison(driver, science_test_data_dir,
+                           compare_data_dict,
+                           result_dir, plot_dir,
+                           plots_to_make):
+    ''' makes snotel figures
+    '''
+
+    # plot settings
+    plot_variables = {'OUT_SWE': 'mm', 'OUT_ALBEDO': 'fraction',
+                      'OUT_SALBEDO': 'fraction', 'OUT_SNOW_DEPTH': 'mm',
+                      'OUT_SNOW_CANOPY': '%', 'OUT_SNOW_PACK_TEMP':
+                      'degrees C', 'OUT_SNOW_MELT': 'mm', 'OUT_R_NET':
+                      '$W/{m^2}$', 'OUT_LATENT': '$W/{m^2}$',
+                      'OUT_SENSIBLE': '$W/{m^2}$'}
+
+    for filename in os.listdir(os.path.join(science_test_data_dir,
+                                            'datasets',
+                                            'snotel',
+                                            'observations')):
+
+        # get lat/lng from filename
+        file_split = re.split('_', filename)
+        lng = file_split[3].split('.txt')[0]
+        lat = file_split[2]
+
+        # loop over data to compare
+        data = {}
+        for key, items in compare_data_dict.items():
+
+            # read in data
+            if key == "snotel":
+                data[key] = read_snotel_swe_obs(filename,
+                                                science_test_data_dir,
+                                                items)
+
+            elif key == "VIC.4.2.d":
+                data[key] = read_vic_42_output(lat, lng,
+                                               science_test_data_dir,
+                                               items)
+
+            else:
+                data[key] = read_vic_5_output(lat, lng,
+                                              science_test_data_dir,
+                                              items)
+
+        # loop over variables to plot
+        for plot_variable, units in plot_variables.items():
+
+            if 'water_year' in plots_to_make:
+
+                with plt.rc_context(dict(sns.axes_style("whitegrid"),
+                                    **sns.plotting_context("notebook"))):
+                    fig, ax = plt.subplots(figsize=(10, 10))
+
+                    df = pd.DataFrame({key: d[plot_variable] for key, d in
+                                      data.items() if plot_variable in d})
+
+                    for key, series in df.iteritems():
+
+                        series.plot(use_index=True,
+                                    linewidth=compare_data_dict[key]
+                                    ['linewidth'],
```

Comment:
I still want this to be fixed.  The `['linewidth'],` should go on the line above. The proper way to handle this is:

```Python
                    for key, series in df.iteritems():
                        series.plot(
                            use_index=True,
                            linewidth=compare_data_dict[key]['linewidth'],
                            ax=ax,
                            color=compare_data_dict[key]['color'],
                            linestyle=compare_data_dict[key]['linestyle'],
                            zorder=compare_data_dict[key]['zorder'])
```